### PR TITLE
Add more stories

### PR DESCRIPTION
**Pull request by marto55**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,21 @@
...
+
+1. As user, I want to be able to sign up and have my own account, so that my chats are kept
+
+2. As user, I want to be able to sign in to my account, so that I can access my chats
+
+3. As user, I want the login page to be simple and pretty, so that I don't get overwhelmed whenever I'm in it
+
+4. As user, I want to be able to toggle between light and dark mode, so that I can protect my eyes and because it's cooler
+
+5. As user, I want to be able to see "click to open" list of my chats after log in, so that I can quickly start chatting
```
Comment (Martincho2003) [inline README.md:13]: U sure that the label of the button will be named like that? And also don't you think that we can make them appear directly or you want to be with dropdown menu or something like that?

File: README.md [hunk 2]
```diff
@@ -1 +1,21 @@
-# Social-network
\ No newline at end of file
+# We are A-Team
+
+## The user stories
+
+1. As user, I want to be able to sign up and have my own account, so that my chats are kept
```
Comment (Martincho2003) [inline README.md]: what about replacing "kept" with "saved and kept in the database?

Comment (marto55) [reply README.md]: the user doesn't know about the database

File: README.md (hunk 1 above)
Comment (marto55) [reply README.md:13]: im almost sure it isn't called like that, just wanted to quickly show my vision

Comment (marto55) [reply README.md:13]: No i don't think the chat should be in a drop down, I thing they should be immediately visible for quicker more seemless experience

File: README.md (hunk 2 above)
Comment (marto55) [reply README.md]: but you are right that the stories are for us and I comply

File: README.md (hunk 1 above)
Comment (Martincho2003) [reply README.md:13]: So this user story must be deleted?

Comment (Martincho2003) [reply README.md:13]: Ohhh I understand the idea now and i like it. Sorry

**Review by Martincho2003 — APPROVED** [review]

**Pull request merged**